app/js/lib/foxbox/services/ip-camera.js

```diff
@@ -14,10 +14,6 @@ export default class IpCameraService extends BaseService {
     return TYPE;
   }
 
-  getImageList() {
-    return this.get('image_list');
-  }
```

Comment:
Like we did with ```turnOn``` and ```turnOff``` for the light service - because it's unused - so less code to maintain until we really need it. But tell me if you want me to return it back.